src/components/Carousel.tsx

```diff
@@ -878,7 +878,7 @@ export default class Carousel extends React.Component<Props, State> {
             containerStyles.height = this.state.itemSize;
         }
         return (
-            <div className={klass.ROOT(this.props.className)} ref={this.setCarouselWrapperRef} tabIndex={0}>
```

Comment:
This will introduce other issues. I don't have the solution for you but keyboard navigation currently is only enabled if the carousel has focus. Removing that will make it not focusable. I'm happy to accept any accessibility improvements but please consider the impact...